start-site/src/main/resources/application.yml

```diff
@@ -142,6 +142,10 @@ initializr:
         mappings:
           - compatibilityRange: "[2.1.0.RELEASE,2.6.0-M1)"
             version: 14.7.5
+          - compatibilityRange: "2.5.x-SNAPSHOT"
```

Comment:
All Spring Boot 2.1.x to 2.6.0-M1 excluded use `14.7.5`.  As such, 2.5.x-SNAPSHOT are also using this version. I believe this range is inaccurate.